ROADMAP.md

```diff
@@ -0,0 +1,171 @@
+# Babel Roadmap
+
+> Not everything is set in stone or has an issue for it! Looking to post these to get more people involved or integrate with other projects.
+
+## Ecosystem
+
+### Remake `compat-table` used in preset-env
+
+> https://github.com/kangax/compat-table could use a remake, ideally work with browser vendors on this
```

Comment:
Is there specific help that would be useful for browser vendors to assist with here? I'd love to see if there's anything we can support to make preset-env + the compat tables easier to rely on over time.

Reply:
I know there is https://github.com/mdn/browser-compat-data which has a javascript section but haven't looked into it that much? I guess it would be nice to be able to query some API from the browser's own status pages to generate data instead of waiting for someone to make a PR? Or if we had a service that ran the test262 tests against browsers we could do it too but that would be a pretty big effort? And maybe we don't need to remake it just change the process of submitting/extraction?

Reply:
Thanks for those details. Vendors maintain their own status pages for features (e.g ChromeStatus) which might get updated more often than the different compat tables. I like the idea of periodically pulling from there in a more automated way, trying to get the MDN tables kept up to date or explore if test262 might be a better path forward. Will have a chat with my team.

cc @nolanlawson @apple-web-evangelist @AdaRoseCannon in case you have ideas too.